### PR TITLE
Post audit events

### PR DESCRIPTION
**Pull request by shriramshankar**

### CODE REVIEW

**Comment by shriramshankar** [comment]
@iamigo @pallavi2209 This PR is for a minor change to the audit table along with some tests to make sure that date comparison will work as expected. I will incoroprate any comments on this PR into my next one.

File: api/v1/swagger.yaml [hunk 1]
```diff
@@ -1330,7 +1330,6 @@ paths:
                     "Collector", "Subject", etc.
               required:
                 - details
```
Comment (shriramshankar) [inline api/v1/swagger.yaml]: What about making details not required and defaulting it to {}; similar to what we did for the "loggedAt" column?

Comment (iamigo) [reply api/v1/swagger.yaml]: OK feel free to make that change here in swagger, adding default value

Comment (shriramshankar) [reply api/v1/swagger.yaml]: OK, will do it.

**Comment by shriramshankar** [comment]
@iamigo @pallavi2209 Anything else that needs changes in the PR?

**Comment by shriramshankar** [comment]
I have included the changes in #621. Can we merge this, if there is nothing else to review ? @iamigo @pallavi2209

**Pull request merged**